src/opt/helm2-deprecation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+YELLOW='\033[1;33m'
+BLUE='\033[1;34m'
+PURPLE='\033[1;35m'
+CYAN='\033[1;36m'
+WHITE='\033[1;37m'
+NO_COLOR='\033[0m'
+
+echo -e ${BLUE}
+echo -e "----------------------------------------------------------------------"
+echo -e "${PURPLE}DEPRECATION NOTICE:${YELLOW} We're replacing helm version 2 by version 3."
+echo -e "  Please, migrate your setup! Helm v3 is incompatible with v2."
+echo -e "  Migration guide: ${WHITE}https://helm.sh/docs/topics/v2_v3_migration/"
+echo -e "${YELLOW}"
+echo -e "  The binary /bin/helm will point to /bin/helm3 starting June 1, 2020."
+echo -e "  Please update calls to 'helm' by 'helm3' before this change."
+echo
+echo -e "  The binary /bin/helm2 will be available until December 1, 2020."
+echo -e "  You may update calls to 'helm' by 'helm2' during that period."
+echo -e ${CYAN}
+echo -e "  Thank you for using APPUiO!"
+echo -e "  Technical support: ${WHITE}support@vshn.ch${BLUE}"
+echo -e "----------------------------------------------------------------------"
+echo -e ${NO_COLOR}
```

Comment:
As long as Helm v2 is officially supported, I think we should keep it. I think this deprecation notice should be about the change of the *default binary `helm`* to symlink to `helm3`, but still leave v2 around and available in `/bin/helm2`.

Also, I think it's better to output to stderr, in case 3rd party tool are parsing the output of helm binary.

As for the date: We then need to make sure all our customer setups we know of should be migrated to helm 3 by then. I'm not sure if that's reasonable in that time.

Reply:
As I'm re-adding image cleanup I also modify this bit